tcs-client/src/main/java/com/transformuk/hee/tis/tcs/client/service/impl/TcsServiceImpl.java

```diff
@@ -698,6 +698,12 @@ public List<GmcDetailsDTO> findGmcDetailsIn(List<String> gmcIds) {
     return responseEntity.getBody();
   }
 
+  public GmcDetailsDTO getGmcDetailsById(Long id) {
```

Comment:
I didn't include a javadoc because there aren't any in this file. I can amend if needed, but then perhaps we need to write javadocs for all the client functions?